### PR TITLE
Add additional save options

### PR DESCRIPTION
**Pull request by ryanpepper**
Hi @prabhuramachandran, I've removed the whitespace changes and created a new pull request.

R

### CODE REVIEW

**Comment by prabhuramachandran** [comment]
Thanks @ryanpepper.  Not sure why the builds are failing.  @itziakos, any ideas?  I would like to merge this.

**Comment by ryanpepper** [comment]
Stupid mistake on my end :)

File: tvtk/plugins/scene/ui/actions.py [hunk 1]
```diff
@@ -62,10 +62,21 @@ def perform(self, event):
...
+        descriptions = [
+            'PNG', 'JPG', 'JPEG', 'TIFF', 'Bitmap', 'PostScript', 'EPS',
+            'TeX', 'RIB', 'WRL', 'Geomview', 'PDF', 'VRML', 'Wavefront',
+	    'Povray', 'X3D'
+        ]
+
+        wildcard = ''
+
+        for description, extension in zip(descriptions, extensions):
+            wildcard += '{} ({})|{}|".format(description, extension, extension)
```
Comment (prabhuramachandran) [inline tvtk/plugins/scene/ui/actions.py]: Another mistake here!

**Comment by prabhuramachandran** [comment]
Thanks!  LGTM, merging.

**Pull request merged**